config/config.example.toml

```diff
@@ -731,18 +731,21 @@ connector_list = ""
 [recipient_emails]
 recon = "test@example.com"
 
+[cell_information]
+id = "12345"
```

Comment:
Can we have documentation comments for these? Please add documentation comments in `config/deployments/env_specific.toml` as well.